Makefile

```diff
@@ -22,6 +22,8 @@ BUILD_LIBC_TOP_HALF ?= yes
 # The directory where we will store intermediate artifacts.
 OBJDIR ?= build/$(TARGET_TRIPLE)
 
+BUILD_WASI64 ?= no
```

Comment:
Maybe `BUILD_WASM64` would better match the target name?

Reply:
what about just WASI64?

Reply:
But WASI64 is not a thing as far as I know.  It doesn't mean much to me as the reader.. whereas WASM64 is the name of the llvm target.

Reply:
but wasm64 is not wasi64, because you also have wasm64-emscripten.

Reply:
> But WASI64 is not a thing as far as I know. It doesn't mean much to me as the reader.. whereas WASM64 is the name of the llvm target.

what about just allowing users to set TRIPLE?

Reply:
Changed to WASM64